workflow/metrics/collector.go

```diff
@@ -69,7 +68,7 @@ func NewWorkflowRegistry(informer cache.SharedIndexInformer) *prometheus.Registr
 // NewTelemetryRegistry creates a new prometheus registry that collects telemetry
 func NewTelemetryRegistry() *prometheus.Registry {
 	registry := prometheus.NewRegistry()
-	registry.MustRegister(prometheus.NewProcessCollector(os.Getpid(), ""))
+	registry.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
```

Comment:
Seems like this breaks behavior? Take a look at https://github.com/argoproj/argo/pull/2052#discussion_r373096063

Reply:
yes - I expect so - need to investigate